src/foam/core/CompoundException.js

```diff
@@ -98,21 +96,28 @@ foam.CLASS({
     {
       name: 'add',
       args: [{ name: 't', javaType: 'Throwable' }],
-      javaCode: 'getExceptions().add(t);'
+      javaCode: 'addSuppressed(t);'
```

Comment:
If it is a CompoundException or just has suppressed exceptions, should we flatten it?

Reply:
Yes, that's better since the nest suppressed exceptions will be surfaced in the log.
Fixed in https://github.com/foam-framework/foam2/pull/4767/commits/ecd62ad8f2e754522ddb7fe90555bf46bb54efe8.